rllib/offline/json_writer.py

```diff
@@ -72,7 +74,7 @@ def write(self, sample_batch):
             len(data), f,
             time.time() - start))
 
-    def _get_file(self):
+    def _get_file(self) -> SerializedType:
```

Comment:
Should be returning a file type.

Reply:
I put FileType=Any in typing. Bc the actual class of files is called: _io.TextIOWrapper